src/components/pages/OnboardPage.tsx

```diff
@@ -64,17 +60,28 @@ const OnboardPage = ({ unmountSelf }: IOnboardPageProps) => {
           </h1>
         </div>
         <div className="mt-10 flex flex-col">
-          <div className="flex flex-col gap-4" onClick={moveToMain}>
-            {mockPorts.map((port) => {
-              return (
+          <div className="flex flex-col gap-4">
```

Comment:
add `overflow-y-auto`; the preview was overflowing for me

Reply:
Added.